backend/src/akvo/lumen/component/tenant_manager.clj

```diff
@@ -1,13 +1,17 @@
 (ns akvo.lumen.component.tenant-manager
   "Component that controll the tenants,
   We use the first domain label e.g. t1 in t1.lumen.akvo.org to dispatch."
-  (:require [akvo.lumen.lib :as lib]
+  (:require [akvo.lumen.component.hikaricp :as hikaricp]
+            [akvo.lumen.lib :as lib]
             [akvo.lumen.lib.aes :as aes]
+            [akvo.lumen.monitoring :as monitoring]
             [akvo.lumen.protocols :as p]
-            [clojure.tools.logging :as log]
-            [integrant.core :as ig]
+            [akvo.lumen.specs.components :refer (integrant-key)]
```

Comment:
Since we used vectors most of the places, is there a reason for using the list notation?